docker-compose.yml

```diff
@@ -2,12 +2,13 @@ x-env: &env
   POSTGRES_DB: accounts
   POSTGRES_USER: accounts
   POSTGRES_PASSWORD: accounts
+  TBA_DEV: yes
 services:
 
   postgres:
     image: postgres:17
     volumes:
-      - db:/var/lib/postgresql/data
+      - ./db:/var/lib/postgresql/data
```

Comment:
Oh this is meant to be a docker volume. We can have it as a physical volume, just maybe punt it to ./postgres_data/accounts_db/ ? (I have another one incoming later for keycloak_db.) 

And then gitignore the postgres_data folder.